EmployeeManagementSystem/BusinessLogicLayer/Mapping/BusinessMappingProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
+
+namespace BusinessLogicLayer.Mapping
+{
+    public class BusinessMappingProfile : Profile
+    {
+        public BusinessMappingProfile()
+        {
+            //Departments
+            CreateMap<BusinessLogicLayer.Models.Department, DataAccessLayer.Entities.Department>().ReverseMap();
```

Comment:
You have two different models with the same name Department. Ty to avoid that. For example, use DepartmentEntity for db entities, and DepartmentViewModel for business layer